client/verta/verta/dataset/entities/_dataset.py

```diff
@@ -321,7 +321,7 @@ def create_version(self, content, desc=None, tags=None, attrs=None, date_created
 
         Parameters
         ----------
-        content : `dataset blob subclass <versioning.html#dataset>`__
+        content : `dataset content <verta.dataset.html>`__
```

Comment:
Broken link can be clicked here: https://verta.readthedocs.io/en/master/_autogen/verta.dataset.entities.Dataset.html#verta.dataset.entities.Dataset.create_version.

It takes you to `_autogen/versioning.html`, but that page no longer exists. `_autogen/verta.dataset.html` is the correct page.